app/views/health/show.html.erb

```diff
@@ -1 +1 @@
-Limber is up!
\ No newline at end of file
+Limber is up!
```

Comment:
Possibly one for another day! But is there a 'Limber is down!’ 404/ 400/ 500 page anywhere? Or would this be something to add to NGINX?